sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
-## 1.2.0-preview.1 (Unreleased)
+## 1.1.1
 
 ### Fixes and improvements
-- Fix `UsernamePasswordCredential` constructor parameter mishandling
+- Fixed `UsernamePasswordCredential` constructor parameter mishandling
+- Updated `ManagedIdentityCredential` endpoint discovery to avoid throwing
+- Fixed 'ManagedIdentityCredential' to raise CredentialUnavailableException on 400 return from the service where no identity has been assigned
+- Updated error messaging from `DefaultAzureCredential`to more easily root cause
```

Comment:
Maybe add a space after the closing backtick and make this read a little nicer by saying something like "to more easily root cause why credentials are not found"